core/src/main/scala/org/locationtech/rasterframes/util/package.scala

```diff
@@ -203,6 +203,25 @@ package object util {
         .mkString("| ", " |\n| ", " |")
       header + body
     }
+
+    def toHTML(numRows: Int = 5, truncate: Boolean = false): String = {
+      import df.sqlContext.implicits._
+      val cols = df.columns
+      val header = "<thead>\n" + cols.mkString("<tr><th>", "</th><th>", "</th></tr>\n") + "</thead>\n"
+      val stringifiers = cols
+        .map(c => s"`$c`")
+        .map(c => df.col(c).cast(StringType))
+        .map(c => if (truncate) substring(c, 1, 40) else c)
```

Comment:
if truncate is true, would like to see a representation that the string has been trunctated. E.g. `Some really really long string about something complicated`

> `Some really really long string about som` 👎 
> `Some really really long string about ...` 👍 

This would be the same in the `toMarkdown` method.

Reply:
See my prior mention of this... https://github.com/locationtech/rasterframes/pull/284#discussion_r314414198